OJP/OJP_ModesSupport.xsd

```diff
@@ -78,12 +108,38 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
-	<xs:simpleType name="ContinuousModesEnumeration">
+	<xs:simpleType name="ContinuousModesOfOperationEnumeration">
 		<xs:annotation>
-			<xs:documentation>Types of cmodes that  run at any time without a timetable.</xs:documentation>
+			<xs:documentation>Types of MODES OF OPERATION that run at any time without a timetable and that are not classic flexible services.</xs:documentation>
 		</xs:annotation>
 		<xs:restriction base="xs:string">
 			<xs:enumeration value="walk"/>
+			<xs:enumeration value="commuterCarPooling">
```

Comment:
Why is this Continuous: there is most often a scheduled pick up time !

Reply:
@Aurige  comes with the pickup being a PLACE and not a SITE ELEMENT. We don't have the tools for that in OJP. So perhaps you want to weight in on that.  Perhaps we need to have some enum values in both enumerations "demandresponsive" for TimedLeg (line based) and ContinuousLeg (for area non-stop based). This might also be true for commutercarPooling.  Should I update accordingly?

Reply:
updated